style.css

```diff
@@ -1,9 +1,14 @@
 #main{
     align-items: center;
-    background-color: rgb(184, 179, 179);
+    background-color: rgba(0, 0, 0, 0.5);
     display: flex;
     flex-direction: column ;
 }
+
+body {
+    color: #fff;
+}
+
 #title{
     color:black;
     align-items: center;
```

Comment:
- Good job! as a recommendation I think it's better to use hexadecimal for assigning color attributes.